### PR TITLE
feat(generate): Added UUID V7 generator

### PR DESCRIPTION
**Pull request by ekkinox**

### CODE REVIEW

File: generate/uuidv7/generator.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// Generate returns a new UUID V7, using [Google UUID].
+//
+// [Google UUID]: https://github.com/google/uuid
+func (g *DefaultUuidV7Generator) Generate() (string, error) {
+	uuid, err := googleuuid.NewV7()
+	if err != nil {
+		return "", err
+	}
+
+	return uuid.String(), nil
```
Comment (ndk) [inline generate/uuidv7/generator.go]: How about returning `uuid.UUID`? Mostly, we're operating with `uuid.UUID` not `string`. :)

Comment (ekkinox) [reply generate/uuidv7/generator.go]: I can, but what do you do with it except using its string representation ?

Comment (ekkinox) [reply generate/uuidv7/generator.go]: ah I see, you mean DB models in structs, ok, updating

Comment (ekkinox) [reply generate/uuidv7/generator.go]: done

**Review by ndk — APPROVED** [review]

**Review by ndk — APPROVED** [review]

**Pull request merged**